### ISSUE TITLE
Control over logging

### ISSUE THREAD

**Issue by bararchy** [Codcore/Amethyst#19]
We need to have more control over logging:

1) Control of output IO (file, STDOUT, STDERR, etc..)
2) Control over format (clear text, Logger, etc..)
3) Control via levels, warn, error, fatal etc ..

**Comment by Codcore** [comment]
@bararchy yes, Amethyst really lacks of logging control. I'm working on it.

**Comment by puppetpies** [comment]
Can you use the AddHandler for Logging control in Amethyst ?
